README.md

```diff
@@ -108,7 +108,7 @@ A test run using the Docker CLI should return usage information and confirm that
           --id string                            identifier for this container
           --log-level string                     log level, must be one of: panic, fatal, error, warn, info, debug (default "debug")
           --percent-failure int                  percentage of requests that this service will automatically fail
-          --sleep-in-millis int                  amount of milliseconds to wait before actually start processing as request
+          --sleep-in-millis int                  amount of milliseconds to wait before actually start processing a request
```

Comment:
Should this help output also include the `--terminate-after` flag that you're adding? Something like:

```
...
          --sleep-in-millis int                  amount of milliseconds to wait before actually start processing a request
          --terminate-after int                  terminate the process after this many requests
...